manifests/vhost.pp

```diff
@@ -2380,6 +2380,19 @@
     }
   }
 
+  if $servername != 'default' and $servername != 'default-ssl' {
+    if $port {
+      $check_http_url = "$servername:$port"
+    } else {
+      $check_http_url = $servername
+    }
+    if ($ssl) {
+        check_http { "https://${check_http_url}": }
```

Comment:
I think the indentation is off by two whitespaces here

Reply:
Thank you @bastelfreak for the quick review.